projects/kubernetes-sigs/image-builder/Makefile

```diff
@@ -103,9 +103,9 @@ else ifeq ($(IMAGE_FORMAT),raw)
 		else
 			RELEASE_TARGETS=
 		endif
-	else ifeq ($(IMAGE_OS),rhel)
+	else ifeq ($(IMAGE_OS),redhat)
 		S3_TARGET_PREREQUISITES=$(FINAL_RAW_IMAGE_PATH)
-		RELEASE_TARGETS=release-raw-rhel-8 upload-artifacts-raw
+		RELEASE_TARGETS=release-raw-$(IMAGE_OS) upload-artifacts-raw
```

Comment:
Are you intentionally dropping the `-8` here?

Reply:
Yes. The idea is to open it up with image builder cli when we do it.